IndicatorFields/incidentfield-threattypes.json

```diff
@@ -1,27 +1,26 @@
 {
     "id": "indicator_threattypes",
     "version": -1,
-    "modified": "2019-12-29T12:15:10.553559972Z",
+    "modified": "2020-02-06T16:01:51.156723531Z",
     "sortValues": null,
-    "propagationLabels": null,
     "vcShouldIgnore": false,
     "commitMessage": "",
     "shouldCommit": false,
     "name": "Threat Types",
     "prevName": "Threat Types",
     "ownerOnly": false,
     "placeholder": "",
-    "description": "Category of threat associated to this indicator. For example, Phishing, Command \u0026 Control, TOR, etc.",
+    "description": "Category of threat associated to this indicator by the feed vendor. For example, Phishing, Command \\u0026 Control, TOR, etc.",
     "cliName": "threattypes",
-    "type": "tagsSelect",
+    "type": "grid",
```

Comment:
will this break bc

Reply:
This field is only from 5.5 so wont break anything